editor/js/editable-wat.js

```diff
@@ -245,6 +245,7 @@ import {
       // using an async function since WebAssembly.instantiate is async and
       // we need to await in order to capture errors
       const AsyncFunction = Object.getPrototypeOf(
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
         async function () {}
       ).constructor;
```

Comment:
Might be a good idea to simplify this while we're here:

```suggestion
      // eslint-disable-next-line @typescript-eslint/no-empty-function
      const AsyncFunction = (async function () {}).constructor;
```

Reply:
This looks good.